components/31-molecules/opening-hours/_opening-hours.scss

```diff
@@ -109,7 +109,7 @@ $week-breakpoint: '(min-width: 750px)';
 
           @include phablet {
             width: $month-day-size;
-            height: $month-day-size / 2;
+            height: $month-day-size *.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.